packages/grafana-ui/src/components/Link/TextLink.story.tsx

```diff
@@ -46,14 +46,14 @@ export const Example: StoryFn = (args) => {
       <StoryExample name="This is a 'inline + external' link with the default behaviour">
         <P>
           To get started with a forever free Grafana Cloud account, sign up at &#160;
-          <TextLink href="https://grafana.com/" external {...args}>
+          <TextLink href="https://grafana.com/" {...args} inline>
             grafana.com
           </TextLink>
```

Comment:
This should be both inline + external, right?

Reply:
Yes, but the `external = true` is in the `args`, as in Storybook we cannot work with internal links.